0.23.0のlinux cpuビルドが落ちる

## 不具合の内容

タイトルのとおりです。

### 現象・ログ

https://github.com/VOICEVOX/voicevox/actions/runs/13576056107/job/37952440084

arm64のエンジンを使おうとしてる？
```
17s
Prepare all required actions
Run ./.github/actions/download-engine
Run TEMPDIR=$(echo '/home/runner/work/_temp' | sed -e 's_\\_/_g')
Run curl -s https://api.github.com/repos/VOICEVOX/voicevox_engine/releases \
Run if [ "linux-cpu" = "" ]; then
7z.txt url: https://github.com/VOICEVOX/voicevox_engine/releases/download/0.23.0/voicevox_engine-linux-cpu-arm64-0.23.0.7z.txt
Files to download:
- voicevox_engine-linux-cpu-arm64-0.23.0.7z.001
Download url: https://github.com/VOICEVOX/voicevox_engine/releases/download/0.23.0/voicevox_engine-linux-cpu-arm64-0.23.0.7z.001, dest: /home/runner/work/_temp/voicevox_engine-linux-cpu-arm64-0.23.0.7z.001

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=C.UTF-8,Utf16=on,HugeFiles=on,64 bits,4 CPUs AMD EPYC 7763 64-Core Processor                 (A00F11),ASM,AES-NI)

Scanning the drive for archives:
1 file, 1428799658 bytes (1363 MiB)

Extracting archive: /home/runner/work/_temp/voicevox_engine-linux-cpu-arm64-0.23.0.7z.001
--
Path = /home/runner/work/_temp/voicevox_engine-linux-cpu-arm64-0.23.0.7z.001
Type = Split
Physical Size = 1428799658
Volumes = 1
Total Physical Size = 1428799658
----
Path = voicevox_engine-linux-cpu-arm64-0.23.0.7z
Size = 1428799658
--
Path = voicevox_engine-linux-cpu-arm64-0.23.0.7z
Type = 7z
Physical Size = 1428799658
Headers Size = 10856
Method = Delta LZMA2:24
Solid = +
Blocks = 3

mv: cannot stat '/home/runner/work/_temp/tmp-extract/linux-cpu/*': No such file or directory
Everything is Ok

Folders: 135
Files: 1091
Size:       1804675860
Compressed: 1428799658
Error: Process completed with exit code 1.
```

## その他

amd64用のビルドなのにarm64のエンジンを使おうとして何らかのエラーが出てる･･････？

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -72,7 +72,7 @@ jobs:
           # Linux CPU
           - artifact_name: linux-cpu-prepackage
             artifact_path: dist_electron/linux-unpacked
-            voicevox_engine_asset_name: linux-cpu
+            voicevox_engine_asset_name: linux-cpu-x64
             package_name: voicevox-cpu
             compressed_artifact_name: voicevox-linux-cpu
             app_asar_dir: prepackage/resources

--- a/public/updateInfos.json
+++ b/public/updateInfos.json
@@ -9,17 +9,21 @@
       "バグ修正"
     ],
     "contributors": [
-      "sevenc-nanashi",
-      "romot-co",
+      "aoirint",
+      "Hiroshiba",
+      "HyodaKazuaki",
+      "jdkfx",
+      "madosuki",
       "MT224244",
       "nanae772",
-      "madosuki",
-      "sabonerune",
       "raa0121",
-      "takusea",
-      "jdkfx",
+      "romot-co",
+      "sabonerune",
+      "sevenc-nanashi",
+      "shunjilin",
       "sigprogramming",
-      "Hiroshiba"
+      "takana-v",
+      "takusea"
     ]
   },
   {

